can-define.js

```diff
@@ -64,7 +64,7 @@ module.exports = define = ns.define = function(objPrototype, defines, baseDefine
 	// for any value that has a default value.
 	replaceWith(objPrototype, "_data", function() {
 		var map = this;
-		var data = {};
+		var data = Object.create(null);
```

Comment:
why is this needed?  (there's another issue where people are having problems with this technique #86)